Signal/src/ViewControllers/ConversationView/Cells/OWSIncomingMessageCell.h

```diff
@@ -0,0 +1,18 @@
+//
+//  Copyright (c) 2017 Open Whisper Systems. All rights reserved.
+//
+
+#import "OWSMessageCell.h"
+
+//#import "OWSExpirableMessageView.h"
+
+NS_ASSUME_NONNULL_BEGIN
+
+// TODO: Remove this class.
+@interface OWSIncomingMessageCell : OWSMessageCell
```

Comment:
It's not clear that we'll need separate classes for "incoming" and "outgoing" message cells, but I haven't combined them yet.